CafePrices/CafePrices/Model/Item.swift

```diff
@@ -6,11 +6,8 @@
 //
 
 import Foundation
-class Item {
-    var itemName: String
-    var itemImage: String
-    init(itemName: String, itemImage: String) {
-        self.itemName = itemName
-        self.itemImage = itemImage
-    }
```

Comment:
why you did this?

Reply:
Sorry for the init delete
And for the view, i´ll do the changes, thanks

Reply:
Also "Detalles..." Was because i´d like it haha